src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -746,13 +746,13 @@
                         </Style>
                     </ListView.ItemContainerStyle>
                 </ListView>
-                <uicontrols:AddonsTreeView Grid.Row="2"
+                <!--<uicontrols:AddonsTreeView Grid.Row="2"
                                            DataContext="{Binding}">
                     <uicontrols:AddonsTreeView.Visibility>
                         <Binding Path="SearchAddonsVisibility"
                                  Converter="{StaticResource BoolToVisibilityCollapsedConverter}" />
                     </uicontrols:AddonsTreeView.Visibility>
-                </uicontrols:AddonsTreeView>
+                </uicontrols:AddonsTreeView>-->
```

Comment:
As I know AddonsTreeView is not used now. Addons and builtin nodes are together. Should AddonsTreeView be presented?